app/controllers/manage_proposal_controller.rb

```diff
@@ -110,6 +109,12 @@ def publish_delete_proposal(proposal, provider)
       if cmr_response.success?
         flash[:success] = I18n.t('controllers.manage_proposals.publish.flash.delete.success')
         Rails.logger.info("Audit Log: Collection with native_id #{proposal['native_id']} was deleted for #{provider} by #{session[:urs_uid]} by proposal with short name: #{proposal['short_name']} and id: #{proposal['id']}.")
+
+        dmmt_response = cmr_client.dmmt_update_proposal_status({ 'draft_type': proposal['draft_type'], 'id': proposal['id'] }, token, request)
+        unless dmmt_response.success?
+          flash[:error] = I18n.t('controllers.manage_proposals.publish.flash.update_proposal_status.error')
+          Rails.logger.info("Audit Log: dMMT did not successfully transition #{proposal['draft_type']} with id #{proposal['id']} to 'done' status from 'approved'.")
```

Comment:
I think this just needs to say somewhere it is a Proposal that did not transition